templates/config.edn

```diff
@@ -343,7 +343,12 @@
  ;;     ;use Percent-encoding for other invalid characters
  :file/name-format :triple-lowbar
 
- ;; specify the format of the filename for journal files
+ ;; Specify the format of the filename for journal files.
+ ;; Warning:
+ ;;   Changing this setting may cause issues if not configured correctly.
+ ;;   After changing this setting, you must reindex.
+ ;; Available values:
+ ;;  - "yyyy-MM-dd" (default)
+ ;;  - "yyyy_MM_dd" 
```

Comment:
According to the code, any date formatter with valid (accepted by FS) characters could be used here.

Reply:
Sounds good. I don't know if there are edge cases but I guess users have to be cautious of that while modifying this config.